modules/price/src/TwigExtension/CommercePriceFormat.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Drupal\commerce_price\TwigExtension;
+
+use Drupal\Core\Template\TwigExtension;
+
+class CommercePriceFormat extends TwigExtension {
```

Comment:
Seems like it might be kinda silly to call it TwigExtension, since it extends a class with that name. What about just dropping "Commerce" and calling it PriceFormat, or something similar?